ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml

```diff
@@ -702,6 +702,14 @@ tests:
   steps:
     workflow: agent-ibmz-sno-e2e
   timeout: 3h0m0s
+- as: ocp-e2e-sno-power
+  cron: 0 4 * * 3
```

Comment:
NIT: this might cause resource contention/failures depending on which job actually runs (its the same cron definition as the 4.15 job above)